Makefile.in

```diff
@@ -590,7 +582,8 @@ ifeq ($(findstring Darwin,$(shell uname -s)),Darwin)
 		-exec install "{}" $(DESTDIR)$(libdir)/xbmc/"{}" \; \
 		-exec printf " -- %-75.75s\r" "{}" \;
 else
-	@find system addons -regextype posix-extended -type f -not -iregex ".*\.git.*" -iregex ".*\.so|.*\.vis|.*\.xbs|.*\.pvr" -exec install -D "{}" $(DESTDIR)$(libdir)/@APP_NAME_LC@/"{}" \; -printf " -- %-75.75f\r"
+	@find system addons -regextype posix-extended -type f -not -iregex ".*\.git.*" -iregex ".*\.so|.*\.so\.[0-9].*|.*\.vis|.*\.xbs" -exec install -D "{}" $(DESTDIR)$(libdir)/@APP_NAME_LC@/"{}" \; -printf " -- %-75.75f\r"
+	@find -L addons -regextype posix-extended -type f -not -iregex ".*\.git.*" -iregex ".*\.so|.*\.so\.[0-9].*" -exec rsync -aRlq "{}" $(DESTDIR)$(libdir)/@APP_NAME_LC@/ \; -printf " -- %-75.75f\r"
```

Comment:
as already mentioned, install don't install symlinks which are required. i.e. pvr addons are loaded by searching for something like pvr.demo.so but the actual lib is named like  pvr.demo.so.12.1.0
pvr.demo.so -> pvr.demo.so.15.0
pvr.demo.so.15.0 -> pvr.demo.so.12.1.0